src/main/java/core/basesyntax/dao/FruitShopDaoImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitShopDaoImpl implements FruitShopDao {
+
+    @Override
+    public void put(String fruit, Integer quantity) {
+        Storage.fruits.put(fruit, quantity);
+    }
+
+    @Override
+    public Integer get(String fruit) {
+        return Storage.fruits.get(fruit);
+    }
+
+    @Override
+    public List<String> getAll() {
```

Comment:
in such method you should return Map, not formed fruits for report, logic below should be do inside of service for the creation of report